opencsp/app/fixed_pattern_deflectometry/lib/FixedPatternSetupCalibrate.py

```diff
@@ -116,6 +117,16 @@ def __init__(
         self._vecs_cams: list[ndarray] = []
         self._dot_intersection_dists: ndarray
 
+    @property
+    def _to_print(self) -> bool:
+        """If verbose printing is turned on"""
+        return self.verbose in [1, 2]
```

Comment:
I like these methods!  Going forward, would it be useful to have a verbosity enum more generally?  Something like:

```python
class SofastVerbosity(enum.Enum):
    NONE = 0 # no output
    PRINT = 1 # text output only
    ALL = 2 # text and plot output
    PLOT = 3 # plot output only
```

Another option: would it be better to have two boolean variables `self._to_print` and `self._to_plot` instead of `self.verbose`?

Reply:
I liked this idea so much I implemented it in FixedPatternSetupCalibrate. Thanks!

Reply:
Nice!